hack/eks

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: terraform enviroment variables need to be set
+INST_PROFILE_NAME=suwang-eks-worker-nodes-1-NodeInstanceProfile-8LL239W4W3GN
+ROLE_NAME=k8s
+
+AWS_CLI="aws"
+TERRAFORM="terraform"
+KUBECTL="kubectl"
+AWS_IAM="aws-iam-authenticator"
+RUN_PATH="$HOME/tmp/terraform-eks"
+CONFIG_PATH=$(dirname $(readlink -f "$0"))/terraform/eks
```

Comment:
`readlink -f` does not work on Mac

Reply:
perhaps I will keep this as we want to handle symlink, but accordingly to https://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac, we can "brew install coreutils --with-default-names"

Reply:
why not just use bash constructs? this is how I have been testing:
```
THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
CONFIG_PATH=$THIS_DIR/terraform/eks
```

Reply:
The reason that I use readlink is that it allows symlink, for instance on my setup I havesuw@ubuntu ~ $ which eks
/usr/local/bin/eks
suw@ubuntu ~ $ ls -al /usr/local/bin/eks
lrwxrwxrwx 1 root root 23 Feb 22 15:40 /usr/local/bin/eks -> /home/suw/terraform/eks

does BASH_SOURCE support symlink?

Reply:
The `BASH_SOURCE` method won't follow links. I don't think that support is needed here, but if you want to keep using `readlink -f`, I suggest you add some logic to verify that the correct version of readlink is installed if the OS is macOS, and print a helpful message if it's not

Reply:
ok, added readlink as cmd/utility prerequisite in eks.